convert_lcnaf_csv_to_jsonl.rb

```diff
@@ -0,0 +1,24 @@
+require "json"
+require "csv"
+
+File.open("subjects_lcnaf.jsonl", "wb") do |jsonl_file|
+  File.open("lcnaf.skos.ndjson").each_line do |line|
+    line_hash = JSON.parse(line)
+    id = line_hash["@context"]["about"]
+    term = ""
+    graph = line_hash["@graph"]
+
+    graph.each do |graph_entry|
+      if graph_entry["@type"] == "skos:Concept"
+        term = graph_entry["skos:prefLabel"]
+      elsif graph_entry["@type"] == "skosxl:Label"
+        term = graph_entry["skosxl:literalForm"]
+      end
+
+      if !term.empty?
+        jsonl_file << {"id": id.to_s, "scheme": "LCNAF", "subject": term.to_s}.to_json + "\n"
+        break
+      end
+    end
+  end
+end
```

Comment:
This is converting an `.ndjson` into `jsonl`, right? (No csv)

I would guess this helps a little since each json is not as big per line as the original ndjson..., but csv would then save more memory on top of that. Using csv might have been fine and take less space. 

I am sorry I probably biased us in favor of .jsonl files. I originally made the choice in these other contexts, because there might have been added nested fields. CSV doesn't give us nested objects per line nicely as opposed to jsonl files. (in my case it might even have been premature looking back on it - vocabularies might introduce nested translated titles, but it's a might...)

Reply:
Oh jeez - thanks. I just copied the original file and quickly changed things without intending to include it at first, but I figured it would be worth having on here. Even better to have it with a good name and no unnecessary libraries.